playbooks/common/openshift-cluster/upgrades/pre/verify_upgrade_targets.yml

```diff
@@ -31,6 +31,13 @@
     register: avail_openshift_version
     when: not openshift.common.is_containerized | bool
 
+  - name: Fail when unable to determine available OpenShift RPM version
```

Comment:
If you move `set_fact` after `Verify containers are available for upgrade`, you can put:
* `set_fact`
* `Check latest available OpenShift RPM version`
* `Fail when unable to determine available OpenShift RPM version`
* `Verify OpenShift RPMs are available for upgrade`

under common block with one `not openshift.common.is_containerized | bool` when condition.